frontend/src/App.tsx

```diff
@@ -110,7 +110,13 @@ class App extends PureComponent<Props, State> {
     this.state = {
       connectionState: ConnectionState.INITIAL,
       elements: {
-        main: fromJS([makeElementWithInfoText("Please wait...")]),
+        main: fromJS([
+          Map({
+            element: makeElementWithInfoText("Please wait..."),
+            metadata: {},
+            reportId: "<null>",
```

Comment:
How about "no report" instead of "<null>"?

"<null>" could be confusing when trying to debug issues :wink: